.gitignore

```
@@ -4,3 +4,4 @@ dist/
 .idea
 .generated-yaml/
 .vscode
+.core
```

```suggestion
.core

```
Another "no newline character" nit

---

committed this change, thanks